crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -93,8 +82,10 @@ impl Extension for LoggerExtension {
             .any(|(_, operation)| operation.node.selection_set.node.items.iter().any(|selection| matches!(&selection.node, Selection::Field(field) if field.node.name.node == "__schema")));
         if !is_schema && self.config.log_request_query {
             info!(
-                target: "async-graphql",
-                "[Query] {}: {}", self.session_id().await, ctx.stringify_execute_doc(&document, variables)
+                query_id = ctx.data_unchecked::<Uuid>().to_string(),
+                session_id = ctx.data_unchecked::<SocketAddr>().to_string(),
+                "[Query] {}",
+                ctx.stringify_execute_doc(&document, variables)
             );
```

Comment:
I did, but I think because I resolved the previous comments you might have not seen it. It does not work because the trait `tracing::Value` is not implemented for `uuid::Uuid`.  
https://github.com/MystenLabs/sui/pull/15696#discussion_r1458613775